tensorflow_datasets/core/download/download_manager.py

```diff
@@ -514,25 +515,32 @@ def callback(path):
       return self._extract(resource)
     return self._download(resource).then(callback)
 
-  def download_kaggle_data(self, dataset_name: str):
+  def download_kaggle_data(self, competition: str):
```

Comment:
I would rename this `competition_or_dataset`, as it doesn;t have to be restricted for competition

Reply:
Yeah, I thought about this and we mention that it can be either in the documentation so I figured it shouldn't really matter. But I'll change this too